src/main/java/gregtech/common/covers/detector/CoverDetectorFluid.java

```diff
@@ -20,7 +20,7 @@
 
 public class CoverDetectorFluid extends CoverBehavior implements ITickable {
 
-    private boolean isInverted;
+    protected boolean isInverted;
```

Comment:
This solution was not what I had in mind, when writing that you should reuse it. In this project this situation is handled by creating getter and setter for this value.

Reply:
Regarding the setter here it already contains code which is specific for setting it from outside of UI (in this case by tool). That code has to be called separately and does not need to be visible from outside.

Reply:
Apologies, I had assumed you were telling me to set `isInverted` as `protected` so that it could be used in the other covers. 

There were already get/set methods for `isInverted` in the advanced detector covers, so I moved that into the base detector covers and set them as `protected`, and then set `isInverted` back to private (https://github.com/GregTechCEu/GregTech/pull/1448/commits/7a118133c2c6c8e4373899d57af56c0e9b5654f0). 

I'm not sure what the purpose of the other logic in `setInverted()` is, you'd have to ask Tech about that.

Reply:
The other `setInverted()` has special handling when you are setting value for GUI element from outside of GUI as you need to synchronize it so it is correctly propagated to clients. Current implementation of leaving that private and offering rest as protected is correct.